tests/topotests/bgp_aggregate_address_zebra_announce/r1/frr.conf

```diff
@@ -0,0 +1,17 @@
+!
+interface lo
+ ip address 10.0.0.1/32
+!
+ip route 10.0.10.1/32 Null0
+!
+router bgp 65000
+  bgp route-id 10.0.0.1
```

Comment:
This is a typo, an invalid command will be logged. Technically, it's not needed at all here.